<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-27T10:50:23 EST" name="Surefire suite" finished-at="2022-02-27T10:50:30 EST" duration-ms="6608">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner@2b4a2ec7]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2022-02-27T10:50:23 EST" name="Surefire test" finished-at="2022-02-27T10:50:30 EST" duration-ms="6608">
      <class name="TestRunner">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:TestRunner@2b4a2ec7]" started-at="2022-02-27T10:50:23 EST" name="setUpClass" finished-at="2022-02-27T10:50:23 EST" duration-ms="290" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper, io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner@2b4a2ec7]" started-at="2022-02-27T10:50:23 EST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2022-02-27T10:50:30 EST" duration-ms="6198" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login with valid email and password"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Optional[Login Test]"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Welcome to TalentTech] but found [Welcome to TalentTek]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Welcome to TalentTech] but found [Welcome to TalentTek]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pageObject.profilePage.verifyWelcomeTextInProfilePage(profilePage.java:24)
at stepDef.profileStep.iShouldBeAbleToSuccessfullyLogin(profileStep.java:17)
at âœ½.I should be able to successfully login(file:///Users/hbhuiyan/Documents/TalentTEK/git/cucumberBDD2022/src/test/java/features/login.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner@2b4a2ec7]" started-at="2022-02-27T10:50:30 EST" name="tearDownClass" finished-at="2022-02-27T10:50:30 EST" duration-ms="90" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
